.storybook/preview.css

```diff
@@ -1,10 +1,3 @@
-@import "../../packages/components/styles/global.css";
-@import "highlight.js/styles/a11y-light.css";
```

Comment:
You may need the highlight styles back for code snippets (maybe can be moved into preview.tsx?)